.github/workflows/build-push-stolostron.yaml

```
@@ -0,0 +1,31 @@
+name: build and push to quay
+
+on:
+  workflow_dispatch:
+  push:  
+    tags:
+      - 'v*' # tags matching v*, i.e. v0.0.1, v1.0.0-rc.0
+
+env:
+  VERSION_TAG: ${{ github.ref_name }}
+  REPOSITORY: quay.io/gatekeeper/gatekeeper
+
+jobs:
+  build:
+    name: Image build and push
+    runs-on: ubuntu-latest  
+
+    steps:
+    - uses: actions/checkout@v4
+      with: 
+        fetch-depth: 0 # Fetch all history for all tags and branches
+
+    - uses: docker/login-action@v2
+      with:
+        registry: quay.io
+        username: ${{ secrets.QUAY_USER }}
+        password: ${{ secrets.QUAY_PASSWORD }}
+
+    - name: build and push
+      run: |
+        REPOSITORY=${{env.REPOSITORY}} VERSION=${{env.VERSION_TAG}} make docker-buildx-release 
```

nit: Missing trailing newline
```suggestion
        REPOSITORY=${{env.REPOSITORY}} VERSION=${{env.VERSION_TAG}} make docker-buildx-release 

```